google-cloud-video_intelligence/.yardopts

```diff
@@ -0,0 +1,8 @@
+--no-private
+--title=Google Cloud Video Intelligence API
+--exclude lib/google/cloud/video_intelligence/v1beta1
```

Comment:
The exclusions in .yardopts appear incorrect. If it is like other packages, it should only include GAPIC and protobuf docs from `doc` dirs, in this case: `google-cloud-video_intelligence/lib/google/cloud/video_intelligence/v1beta1/doc`. I will update the file. 

I will also add `video_intelligence` to `gcloud-ruby/docs/manifest.json`, in order for the docs to be loaded and accessible via top navigation in the gh-pages site. This will **not** add the docs to the `google-cloud` meta-package.